### ISSUE TITLE
OpenBSD: tor package installation fails

### ISSUE THREAD

**Issue by nusenu** [nusenu/ansible-relayor#85]
Error:

```
    "failed": true, 
    "invocation": {
        "module_args": {
            "build": false, 
            "name": "['tor']", 
            "ports_dir": "/usr/ports", 
            "state": "present"
        }
    }, 
    "item": [
        "tor"
    ], 
    "msg": "Can't find [tor]\n"
```

bug introduced with:
https://github.com/nusenu/ansible-relayor/commit/550d656a2c5ec8f2b588b32d9cf256aab2743bb7

**Comment by nusenu** [comment]
https://groups.google.com/d/msgid/ansible-project/eb9fb1ec-b033-8aa4-c296-97e688439678%40openmailbox.org

**Comment by nusenu** [comment]
Actually this commit introduced the problem:

https://github.com/nusenu/ansible-relayor/commit/06eaad05443f5282b4ac74af688a5f6e60e45b83#diff-2444ad0870f91f17ca6c2a5e96b26823

**Comment by nusenu** [comment]
https://groups.google.com/d/msgid/ansible-project/08037a9f-f46a-e5b0-af75-28f9bf241154%40openmailbox.org

**Comment by nusenu** [comment]
https://github.com/ansible/ansible-modules-core/issues/5708

**Comment by nusenu** [comment]
To workaround this bug when targeting OpenBSD hosts, set ANSIBLE_SQUASH_ACTIONS:
`ANSIBLE_SQUASH_ACTIONS="yum,apt" ansible-playbook ...`

**Comment by bcoca** [comment]
so the setting in ansible.cfg that controls 'squash' was updated after openbsd_pkg got the ability to process lists of packages as well as single packages. This problem would arise from using an older version of openbsd_pkg that did not support this.

**Comment by nusenu** [comment]
Does not work for me on ansible version v2.2.1.0 (currently latest stable version)
without the workaround mentioned above.

Waiting for v2.3 as mentioned in https://github.com/ansible/ansible-modules-core/issues/5708#issuecomment-262540026

**Comment by bcoca** [comment]
the long term fix is making the 'workaround' default.

**Comment by nusenu** [comment]
That is what I assumed, but when will it be released? (v2.3 still correct?)
Are you saying that it should not fail on v2.2.1.0 even without the workaround?

If v2.2.1 is expected to fail: 
looking forward to the release containing the fix (waiting since quite some time for it)
do you have an ETA?
but https://github.com/ansible/ansible/issues/14829 is certainly more annoying/pressing.

I appreciate you coming here.

**Comment by nusenu** [comment]
ansible 2.3.0 has been released and should contain the fix.
